docs/developer/release/prepare.md

```diff
@@ -0,0 +1,42 @@
+# Prepare build environments
+
+## macOS
+
+### Initial setup
+
+- Ensure that the build machine has:
+  - Apple-trusted `Developer ID Application: Freedom of the Press Foundation (94ZZGGGJ3W)` code-signing certificates installed
+- Apple account must have:
+  - A valid application password for `notarytool` in the Keychain. You can verify
```

Comment:
I think this session could be refactored to have the CLI lines in line, and probably some of the explanations as comments, as you did earlier for other steps. Right now, having these as bits to copy-and-paste inline, I find it a bit more difficult to use (nitpick, but I think it makes a difference when releasing)

Reply:
So, the thing with the "Initial setup" sections is that these instructions take place once, and you don't have to run them again for subsequent releases. Also, they are not picked by `generate-release-tasks.py`, so it's not that important to be in a single line. Did you have to copy-paste any of these instructions for the last release though?

Reply:
I trust your instinct, and changed it in cff1db5584507878fab87f3b1b9a1636a0122a37.